### ISSUE TITLE
Unfortunately unusably slow

### ISSUE THREAD

**Issue by MostHated** [CGCookie/retopoflow#690]
Hey there,
I was super excited to use this, but when I fired it up, it is taking several minutes at a time to perform any sort of action. It gave me this to send to you.

Details:

- RetopoFlow: 2.0.2
- Blender: 2.79.6 blender2.7 2019-01-04
- Platform: Windows, 10, 10.0.17763, AMD64, AMD64 Family 23 Model 8 Stepping 2, AuthenticAMD
- GPU: NVIDIA Corporation, GeForce GTX 1050 Ti/PCIe/SSE2, 4.6.0 NVIDIA 417.35, 4.60 NVIDIA
- Target: verts:131333, edges:394018, faces:262694
- Sources: number:1, verts:964, edges:2880, faces:1920
- FPS: current:0.005173, threshold:5, time:10

Thanks,
-MH

**Comment by vxlcoder** [comment]
@MostHated, I think your source and target meshes are swapped. When you start RF, either have nothing selected (creates new target mesh), or select the low-res retopo mesh (for continuing retopoing). See the quick start guide for more details. Let me know if you continue having problems.

**Comment by MostHated** [comment]
My apologies! I did as you mentioned and now it seems to be working. Thank you for that! 

Side question, is there a public roadmap or anything I could keep an eye on now and then to see what the plans are for this, such as if it's coming to 2.80, what new features are coming, etc? From just the few seconds I played with it so far it seems amazing. If it is, in fact, coming for 2.80, I will definitely be purchasing it from the Blender Market.
